src/lib/variants.test.ts

```diff
@@ -118,6 +131,100 @@ describe('variant filters', () => {
         expect(got3).toBe(true);
     });
 
+    it('should filter by user cohort', () => {
+        const now = new Date('2020-03-30T18:00:00');
+        const lessThanThreeMonthsAgo = new Date('2020-03-28T10:30:00');
+        const moreThanThreeMonthsAgo = new Date('2019-12-12T10:30:00');
```

Comment:
What do you think about deriving `lessThanThreeMonthsAgo` and `moreThanThreeMonthsAgo` from `now`? (i.e. subtracting an amount of time from `now` to get them). Just because there are then fewer dates to mentally parse (and less scope to somehow change one of those dates in future to something which _isn't_ what the variable name says).

Reply:
Cool, good shout